src/new-client/src/app/site/[site]/details/edit-information-for-citizens/edit-information-for-citizens-page.tsx

```diff
@@ -0,0 +1,32 @@
+import { fetchInformationForCitizens } from '@services/appointmentsService';
+import AddInformationForCitizensForm from './add-information-for-citizens-form';
+
+type Props = {
+  site: string;
+  permissions: string[];
+};
+
+export const EditInformationForCitizensPage = async ({ site }: Props) => {
+  const informationForCitizens = await fetchInformationForCitizens(
+    site,
+    'site_details',
+  );
+  const siteInformation = informationForCitizens.find(
+    i => i.id === 'site_details/info_for_citizen',
+  );
+
+  return (
+    <>
+      <div className="nhsuk-form-group">
```

Comment:
There is a `<FormGroup/>` component that accepts a legend and a validation error. It could be expanded to accept a hint too if you wanted to use it here but I'm happy either way :)